tests/libnet/hotplug.go

```diff
@@ -123,7 +124,8 @@ func interfaceStatusFromInterfaceNames(queueCount int32, ifaceNames ...string) [
 			InterfaceName: fmt.Sprintf("eth%d", i+initialIfacesInVMI),
 			InfoSource: vmispec.NewInfoSource(
 				vmispec.InfoSourceDomain, vmispec.InfoSourceGuestAgent, vmispec.InfoSourceMultusStatus),
-			QueueCount: queueCount,
+			QueueCount:       queueCount,
```

Comment:
If this change is not mandatory please more this change to different PR.

Reply:
Without this change, the hotplug tests will fail.